src/domain/application/applicationView/ApplicationView.tsx

```diff
@@ -192,9 +191,9 @@ function ApplicationView({ application, hanke, signedInUser, onEditApplication }
                 theme="coat"
                 iconLeft={<IconEnvelope aria-hidden="true" />}
                 onClick={onSendApplication}
-                isLoading={applicationSendMutation.isLoading}
+                isLoading={applicationSendMutation.isLoading || isSendButtonDisabled}
```

Comment:
Send button stays in loading state even after sending is successful because `isSendButtonDisabled` is not set to `false`.

Reply:
Good call, thanks! I added setting disabled to false here and in ApplicationCancel as well. Well, actually, it is not needed in cancelling success because user is redirected to the application list in that case. I took it away from there.